nav2_behavior_tree/plugins/condition/globally_updated_goal_condition.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 Joshua Wallace
+//
```

Comment:
Both nodes need entries in the README in this package to explain what they are and what they do.

Also, in the https://navigation.ros.org/plugins/index.html and https://navigation.ros.org/migration/Galactic.html and https://navigation.ros.org/configuration/packages/configuring-bt-xml.html

Reply:
The current README in the nav2_behavior_tree does not have any documentation about specific BT nodes. I am assuming you would like me to start a new section?

Reply:
Also added to the documentation and submitted a pull request https://github.com/ros-planning/navigation.ros.org/pull/256

Reply:
Just website is fine then, sorry my notebook on boilerplate responses is outdated :laughing: